<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptWidget</class>
 <widget class="QWidget" name="ScriptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/16x16_ico_png/script.png</normaloff>:/16x16_ico_png/script.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="find">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Replace text</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="1">
               <widget class="QLineEdit" name="replaceText"/>
              </item>
              <item row="3" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="btnFindNext">
                  <property name="text">
                   <string>Find</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnReplaceNext">
                  <property name="text">
                   <string>Replace</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnReplaceAll">
                  <property name="text">
                   <string>Replace all</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnFindClose">
                  <property name="text">
                   <string>Close</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="findText"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Find</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QCheckBox" name="findBackwards">
                  <property name="text">
                   <string>Find backwards</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="findCaseSensitive">
                  <property name="text">
                   <string>Case sensitive</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="findWholeWords">
                  <property name="text">
                   <string>Whole words</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="findResult">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Replace with</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="gui::TextEditor" name="code">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="filter">
         <property name="placeholderText">
          <string>Find help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="helpSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTableView" name="tableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>21</number>
          </attribute>
         </widget>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="markdown">
           <string notr="true"/>
          </property>
          <property name="html">
           <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextBrowserInteraction</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/save.png</normaloff>:/32x32_ico_png/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/16x16_ico_png/workspace_open.png</normaloff>:/16x16_ico_png/workspace_open.png</iconset>
   </property>
   <property name="text">
    <string>Open Externally</string>
   </property>
   <property name="toolTip">
    <string>Open in an external application</string>
   </property>
  </action>
  <action name="actionFindText">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/find.png</normaloff>:/32x32_ico_png/find.png</iconset>
   </property>
   <property name="text">
    <string>Find Text</string>
   </property>
   <property name="toolTip">
    <string>Find text in document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplaceText">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/32x32_ico_png/text_replace.png</normaloff>:/32x32_ico_png/text_replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace Text</string>
   </property>
   <property name="toolTip">
    <string>Replace text in document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFindHelp">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/16x16_ico_png/help.png</normaloff>:/16x16_ico_png/help.png</iconset>
   </property>
   <property name="text">
    <string>Find Help</string>
   </property>
   <property name="toolTip">
    <string>Find help in API documentation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::TextEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>findText</tabstop>
  <tabstop>replaceText</tabstop>
  <tabstop>findBackwards</tabstop>
  <tabstop>findCaseSensitive</tabstop>
  <tabstop>findWholeWords</tabstop>
  <tabstop>btnFindNext</tabstop>
  <tabstop>btnReplaceNext</tabstop>
  <tabstop>btnReplaceAll</tabstop>
  <tabstop>btnFindClose</tabstop>
  <tabstop>code</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>findText</sender>
   <signal>returnPressed()</signal>
   <receiver>btnFindNext</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>906</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
